en/docs/administer/blue-green-data-encryption-keyrotation.md

```diff
@@ -0,0 +1,252 @@
+4.  Encryption of sensitive deployment level configuration data in configuration files
+
+With WSO2 IS 5.11.0 onwards symmetric encryption mechanism is used to encrypt internal sensitive runtime data above. The DEK used to encrypt these data are configured in the `deployment.toml` file and it is protected by a KEK. Secure vault is being utilized as of now to protect this DEK. Here we will consider only the rotation of the Data Encryption Key configured in the `deployment.toml` file. 
+
+## DEK rotation frequency
+
+In the case of a security compliance requirement we can see that symmetric DEK rotation can be done in 2 years or less based on the volume of data we have.
+
+In a security breach scenario we must rotate the DEK immediately and re-encrypt all the data to the new DEK.
+
+## Key Rotation Approach
+
+WSO2 IS has introduced an external tool that re-encrypts internal data after rotation of the configured symmetric data encryption key. Here mainly we have considered the re-encryption of the Identity and registry databases and some configuration files. Apart from that the tool will sync end user data that gets generated in the live system to the new setup.
+
+The external java client that performs the above tasks can be created by following the steps [here](../../administer/blue-green-data-encryption-keyrotation/#how-to-create-the-key-rotation-tool).
+
+## Tables that support re-encryption 
+
+At the moment below tables are supported for re-encryption
+
+??? info "Tables that support re-encryption"
+
+    | **Table**                                | **Fields**                                                                      |
+    |------------------------------------------|---------------------------------------------------------------------------------|
+    | IDN_IDENTITY_USER_DATA                   | TOTP secretKey and verifiedSecretKey claims                                     |
+    | IDN_OAUTH2_AUTHORIZATION_CODE            | OAuth2 authorization codes                                                      | 
+    | IDN_OAUTH2_ACCESS_TOKEN                  | OAuth2 access and refresh tokens                                                | 
+    | IDN_OAUTH_CONSUMER_APPS                  | Consumer secrets                                                                | 
+    | WF_BPS_PROFILE                           | BPS profile password                                                            | 
+    | WF_REQUEST                               | WF request credentials                                                          |   
+    | REG_PROPERTY                             | Keystore passwords, Keystore privatekeyPass and Entitlement subscriberPasswords | 
+
+## Configuration files that support re-encryption
+
+At the moment below configuration files are supported for re-encryption
+
+??? info "Configuration files that support re-encryption"
```

Comment:
Shall we make it a normal info box?

Reply:
Sure, addressed with https://github.com/wso2/docs-is/pull/2555/commits/e4826286f4b66bb74b49890f19dce24148b95808